ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -57,6 +53,15 @@ else
 
 fi
 
+#to support vsphere-ipi-zones-multisubnets-external-lb-f28 and vsphere-ipi-zones-multisubnets-external-lb-f28-destructive.add these var to specify the portgroups.
+if [[ -n "${VSPHERE_MULTIZONE_LEASED_RESOURCE:-}" ]]; then
+    multizone_leased_resource1=$(awk '{print $1}' <(echo "${VSPHERE_MULTIZONE_LEASED_RESOURCE}"))
+    multizone_leased_resource2=$(awk '{print $2}' <(echo "${VSPHERE_MULTIZONE_LEASED_RESOURCE}"))
+    multizone_vlanid1=$(awk -F. '{print $3}' <(echo "${multizone_leased_resource1}"))
+    multizone_vlanid2=$(awk -F. '{print $3}' <(echo "${multizone_leased_resource2}"))
+    multizone_portgroup1="ci-vlan-${multizone_vlanid1}"
```

Comment:
can we make a loop for `VSPHERE_MULTIZONE_LEASED_RESOURCE` to create some dynamic var name, after all, `VSPHERE_MULTIZONE_LEASED_RESOURCE` can be 1, or 3 and others, to make the code to be more robust.

Reply:
I did some local testing, maybe here we can reuse the logic:
```
[jianlinliu@jialiu-laptop rehearse]$ for i in aaa bbb; do eval v_${i}='${i}'; done
[jianlinliu@jialiu-laptop rehearse]$ echo $v_aaa
aaa
[jianlinliu@jialiu-laptop rehearse]$ echo $v_bbb
bbb
```